esp/smc/SMCLib/TpWrapper.cpp

```diff
@@ -2050,4 +2050,75 @@ extern TPWRAPPER_API IStringIterator* getContainerTargetClusters(const char* pro
         ret->append_unique(targetName);
     }
     return ret.getClear();
+}
+
+class CContainerWUClusterInfo : public CSimpleInterfaceOf<IConstContainerWUClusterInfo>
```

Comment:
it's ok to use the existing interface (IConstWUClusterInfo) for now, as long as it's not using and/or extending the environment implementation.
This implementation should implement IConstWUClusterInfo, but with most of the methods unimplemented.
Then all the rest of the code is the service is common.

Reply:
Will change